Content.Server/GameObjects/Components/Atmos/GasCanisterComponent.cs

```diff
@@ -327,12 +320,21 @@ private void ToggleValve()
         /// <param name="frameTime"></param>
         public void Update(in float frameTime)
         {
-
             if (ValveOpened)
             {
                 var tileAtmosphere = Owner.Transform.Coordinates.GetTileAtmosphere();
-                Air.ReleaseGasTo(tileAtmosphere.Air, ReleasePressure);
-                _gridAtmosphere.Invalidate(tileAtmosphere.GridIndices);
+                if (tileAtmosphere != null)
```

Comment:
Maybe if it is null, "delete the gas" as if we were in space?

Reply:
Why delete the gas? Just having it leak out is better.

Reply:
There is no place for it to leak to. Therefore, it leaking is it being deleted. Anyway, I've done that, so...